commons/passive/phone/phone_user_interaction.avsc

```diff
@@ -10,7 +10,7 @@
       "name": "PhoneInteractionState",
       "type": "enum",
       "doc": "Interaction type with an Android device.",
-      "symbols": ["STANDBY", "UNLOCKED", "SHUTDOWN", "BOOTED", "OTHER"]
-      }, "doc": "What interaction state the phone has." }
+      "symbols": ["STANDBY", "UNLOCKED", "SHUTDOWN", "BOOTED", "OTHER", "UNKNOWN"]
+      }, "doc": "What interaction state the phone has.", "default": "UNKNOWN" }
```

Comment:
is this default required?

Reply:
"Shouldn't I 😅? Shall I undo these changes?"

Reply:
It depends if they are provided by Android?

Reply:
Fine, Updated It.